.github/workflows/trunk-ci.yaml

```diff
@@ -0,0 +1,26 @@
+name: Pull Request
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - main
+concurrency:
+  group: ${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+permissions: read-all
+
+jobs:
+  trunk_check:
+    name: Trunk Code Quality Runner
+    runs-on: ubuntu-latest
+    permissions:
+      checks: write # For trunk to post annotations
+      contents: read # For repo checkout
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Trunk Code Quality
```

Comment:
Do we have a Trunk account? I'm fine with adding trunk if that is a tool the company wants to adopt

Reply:
trunk doesn't require an account, trunk-the-tool knows its running in CI and doesn't need any interaction with trunk-the-service. trunk accounts are optional and we don't need the paid account, I think. 
I don't think the company needs to adopt it for us to use it in this repo.